packages/fxa-react/components/TextArea/index.tsx

```diff
@@ -0,0 +1,118 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React, {
+  KeyboardEvent,
+  FocusEvent,
+  ChangeEvent,
+  useState,
+  useCallback,
+  useEffect,
+  ReactElement,
+} from 'react';
+import { v4 as uuidv4 } from 'uuid';
+
+export type TextAreaProps = {
+  defaultValue?: string | number;
+  disabled?: boolean;
+  id?: string;
+  label: string;
+  onBlur?: (event: FocusEvent<HTMLTextAreaElement>) => void;
+  onChange?: (event: ChangeEvent<HTMLTextAreaElement>) => void;
+  onFocus?: (event: FocusEvent<HTMLTextAreaElement>) => void;
+  onKeyPress?: (event: KeyboardEvent<HTMLTextAreaElement>) => void;
```

Comment:
Why do we need `onKeyPress` here?

Reply:
Per our Slack chat, I wanted to future-proof some of the more common uses of something like a text input field. Something like listening to key strokes is pretty common. No, we don't need it right now. My preference for this is because I don't like having to introduce foundational, component-wide changes before a specific feature that utilizes the component can be implemented as it could add more development/testing time and throws off the scope of a PR. However, I'm not willing to die on this hill so I'll remove all unnecessary event listeners across all these new components. We also don't need this TextArea altogether right now (it's not in the Figma doc), so I'll remove it.